src/frontend/js/components/DashboardCreditCardsManagement/_styles.scss

```diff
@@ -0,0 +1,135 @@
+.dashboard-credit-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+
+  &__empty {
+    text-align: center;
+    color: r-theme-val(dashboard-credit-cards, empty-color);
+    margin: 0;
+  }
+}
+
+.dashboard-credit-card {
+  &__details {
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+      gap: 1rem;
+    }
+
+    img {
+      width: 83px;
+      height: 50px;
+      object-fit: contain;
+    }
+
+    p {
+      margin: 0;
+      font-size: 18px;
+      color: r-theme-val(dashboard-credit-card-box, base-color);
+    }
+  }
+
+  &__data {
+  }
+
+  &__brand {
+    font-weight: bold;
+
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  &__buttons {
+    display: flex;
+    gap: 0.5rem;
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+    }
+  }
+
+  &__meta {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+    }
+
+    p.dashboard-credit-card__meta__separator {
+      color: r-theme-val(dashboard-box, base-border);
+
+      @include media-breakpoint-down(sm) {
+        display: none;
+      }
+    }
+  }
+
+  h6 {
+    font-size: 18px;
+    color: r-theme-val(dashboard-credit-card-box, title-color);
+    font-weight: bold;
+    margin-bottom: 1rem;
+
+    @include media-breakpoint-down(sm) {
+      text-align: center;
+    }
+  }
+
+  p.dashboard-credit-card__expiration--soon {
+    color: r-theme-val(dashboard-credit-card-box, expire-soon-color);
+  }
+}
+
+.dashboard-edit-credit-card {
+  &__sensitive {
+    display: flex;
+    gap: 1rem;
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+      gap: 0;
+      margin-bottom: 2rem;
+
+      .form-field {
+        flex-grow: 1;
+      }
+    }
+
+    .form-field:first-child {
+      flex-grow: 1;
+      flex-shrink: 0;
+    }
+
+    &__sub {
+      display: flex;
+      gap: 1rem;
+    }
+  }
+}
+
+.credit-card-brand-logo {
+  background-color: r-theme-val(credit-card-brand-logo, background-color);
+  border-radius: $border-radius;
+  height: 50px;
+  width: 83px;
+  padding: 10px;
+
+  &:hover img {
+    transform: rotateY(360deg);
```

Comment:
Are you really sure ? 😂

Reply:
![tadaaa-surprise](https://user-images.githubusercontent.com/9628870/196388526-e0de4d73-b694-4ac4-ac3d-4628b0af8fc3.gif)